assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/block.tsx

```diff
@@ -24,7 +19,10 @@ const Block = ( {
 	return (
 		<TotalsWrapper className={ className }>
 			<TotalsShipping
-				showCalculator={ isShippingCalculatorEnabled }
+				showCalculator={ getSetting(
```

Comment:
Applying this suggestion prevents a TS error

```suggestion
				showCalculator={ getSetting< boolean >(
```